ml_genn/model.py

```diff
@@ -167,7 +167,7 @@ def evaluate_batched(self, data, time, save_samples=[]):
         """Evaluate the accuracy of a GeNN model
 
         Args:
-        data          --  an (x, y) batch dataset
+        data          --  an [[in_1, ..., in_n], [out_1, ..., out_n]] batch dataset
```

Comment:
maybe you should use e.g. ``m`` and ``n`` here rather than ``n`` for both and clarify that ``m`` is the number of inputs in the model and ``n`` is the number of outputs